spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/reactor/ReactorSleuth.java

```diff
@@ -41,59 +40,6 @@
 
 	private static final Log log = LogFactory.getLog(ReactorSleuth.class);
 
-	/**
```

Comment:
This looks like a breaking change... First, why have you removed it? Second if we can remove it, maybe just deprecate it first?

Reply:
SpanSubscriber was used only for lastOperation hook, which is not used anymore. It has package private access level. So it shouldn't be used out of sleuth project. But if you wish I can revert and deprecate it.